### PR TITLE
change the Spark series model to the official name

### PR DESCRIPTION
**Pull request by AAEE86**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] Please open an issue before creating a PR or link to an existing issue
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by AAEE86** [comment]
issues #8075

**Review by crazywoola — CHANGES_REQUESTED** [review]
You can change the label into the new name instead of renaming all things, if. you change those files, people use old configuration will encounter errors.

So for newly added changes, you need to mark old ones to deprecated first and create new files based on that.

**Comment by AAEE86** [comment]
> You can change the label into the new name instead of renaming all things, if. you change those files, people use old configuration will encounter errors.
> 
> So for newly added changes, you need to mark old ones to deprecated first and create new files based on that.

OK

File: api/core/model_runtime/model_providers/spark/llm/Spark Lite.yaml
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark Lite.yaml:1]: Filename should be spark-lite.yaml, do not use blank spaces in file system.

File: api/core/model_runtime/model_providers/spark/llm/Spark Lite.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+model: Spark Lite
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark Lite.yaml]: This should be spark-lite as well.

File: api/core/model_runtime/model_providers/spark/llm/Spark Max.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+model: spark-3.5
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark Max.yaml]: See above comments.

File: api/core/model_runtime/model_providers/spark/llm/Spark Pro.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+model: Spark Pro
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/spark/llm/Spark Pro.yaml]: See above comments.

**Review by crazywoola — CHANGES_REQUESTED** [review]
See comments

**Comment by AAEE86** [comment]
It should be correct. 

It seems that I am not suitable for writing code.😂

**Pull request closed without merging**